### PR TITLE
Add actions-tagger

### LINKED ISSUE #28: bug(ci): Not using latest release action with "v1"
#### Issue/Bug description:

When running the community action maven release workflow in the spring-zeebe repository, all workflows successfully build with the exception of `example/spring-zeebe-example-worker ................ FAILURE [  1.578 s]`

This failed workflow run can be viewed in more detail at: https://github.com/camunda-community-hub/spring-zeebe/runs/4372925957?check_suite_focus=true

Full error text: `Error:  Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:deploy-staged (default-cli) on project spring-zeebe-example-worker: Failed to deploy artifacts: Could not transfer artifact io.camunda:spring-zeebe-example-starter:jar:1.2.7-20211130.202345-1 from/to central (https://oss.sonatype.org/content/repositories/snapshots/): authorization failed for https://oss.sonatype.org/content/repositories/snapshots/io/camunda/spring-zeebe-example-starter/1.2.7-SNAPSHOT/spring-zeebe-example-starter-1.2.7-20211130.202345-1.jar, status: 403 Forbidden -> [Help 1]`

The 'status: 403 Forbidden' has led me to question if the `settings.xml` are correct for `example/spring-zeebe-example-worker`, as they may need to be updated due to the migration of Maven artifacts to Artifactory, particularly if they were/are stored on a local machine. 

The latest release action version `v1.0.9` does not persist unless added manually, and continues to default to using `@v1` as seen in https://github.com/camunda-community-hub/spring-zeebe/commit/bfb4677ac8c6bd0b470524b5a08c0109e47a619e which leads to the above 403 error and failing workflow.

#### Steps to reproduce the issue/bug

1.  Run workflow as intended to automatically release to Maven Central using latest version `v1.0.9` of the Community Action Maven Release tool

#### What's the expected result, or what did you expect to happen?

- All builds complete successfully, using `v1.0.9` of the Community Action Maven release

#### What's the actual result?

- All builds are successful with the exception of `example/spring-zeebe-example-worker ................ FAILURE [  1.578 s]`, this causes the entire workflow run to fail.

#### Platform details
TBD

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Chrome
 - Version: GitHub IDE

#### Additional details/clarification questions

Suggest confirming that the account in question has the privileges needed in Artifactory/Maven central to reach Maven. Also suggest confirming `settings.xml` is configured correctly for the failing build. If needed, suggest creating an API key in Artifactory and updating local maven m2 settings.xml.

Is it possible we have exceeded a quota with Maven? I wouldn't think that is the case, but might be worth looking into?

Data Source[s]: https://stackoverflow.com/questions/59686900/maven-nexus-issue-authorization-failed-for-403-forbidden

https://stackoverflow.com/questions/65663325/403-forbidden-when-trying-to-install-arcgis-java-dependency-with-maven

https://github.com/jitpack/jitpack.io/issues/3388

https://github.com/sonatype-nexus-community/nexus-repo-github-action/issues/4

### PR DESCRIPTION
**Pull request by celanthe**
Added https://github.com/marketplace/actions/actions-tagger to the community action maven release to test if this will automate tagging to `latest` to hopefully resolve #28

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -113,6 +113,8 @@ runs:
             https://api.github.com/repos/${GITHUB_REPOSITORY}/code-scanning/sarifs \
             -d '{"commit_sha":"${GITHUB_SHA}","ref":"${GITHUB_REF}","sarif": "${COMPRESSED_SARIF}"}' || true
         fi
+    - name: Actions Tagger
+      uses: Actions-R-Us/actions-tagger@v2.0.2
```
Comment (berndruecker) [inline action.yml:117]: According to https://github.com/Actions-R-Us/actions-tagger#publish_latest_tag we would need to set this attribute - or not?
`
      - uses: Actions-R-Us/actions-tagger@latest
        with:
          publish_latest_tag: true
`

Comment (celanthe) [reply action.yml]: Hi @berndruecker! That's correct, apologies. I updated the workflow in my latest PR to have the correct attributes as noted in the action-tagger's documentation.

File: action.yml [hunk 2]
```diff
@@ -113,8 +113,17 @@ runs:
...
-    - name: Actions Tagger
-      uses: Actions-R-Us/actions-tagger@v2.0.2
+     name: actions tagger
+      on:
+        release:
+          types: [published, edited]
+        jobs:
+        actions-tagger:
+            runs-on: windows-latest
+        steps:
```
Comment (berndruecker) [inline action.yml:123]: the formatting looks wired for me here locally - but might just look like this in the web.

File: action.yml [hunk 3]
```diff
@@ -113,8 +113,17 @@ runs:
...
         fi
-    - name: Actions Tagger
-      uses: Actions-R-Us/actions-tagger@v2.0.2
+     name: actions tagger
+      on:
+        release:
+          types: [published, edited]
+        jobs:
+        actions-tagger:
+            runs-on: windows-latest
```
Comment (berndruecker) [inline action.yml:122]: Do we need this? Isn't that the default? Then we could save some lines

Comment (celanthe) [reply action.yml]: Removed this! :)

File: action.yml (hunk 2 above)
Comment (celanthe) [reply action.yml]: Removed this line :)

**Review by celanthe — COMMENTED** [review]
Updated action-tagger formatting to reflect changes suggested

**Review by celanthe — COMMENTED** [review]
Updated formatting of line 117

**Comment by celanthe** [comment]
Hi @berndruecker! :) I managed to debug this, made the changes you suggested, and figured out why the pre-commit was failing. I had to update the formatting on line 117 to allow for nested mappings by adding quotes. The pre-commit check passed now, so this should be okay to approve. 

Once that's done, I can cut a new release and tag it with `latest`

**Comment by berndruecker** [comment]
ok

**Pull request merged**